tests/integration/test_join.py

```diff
@@ -329,11 +329,11 @@ def test_intersect(c):
     actual_df = c.sql(
         """
     select count(*) from (
-    select * from df_simple
+    select a, b from df_simple
     intersect
-    select * from df_simple
+    select a, b from df_simple
     intersect
-    select * from df_wide
+    select a, b from df_wide
     ) hot_item
     limit 100
```

Comment:
This query was invalid. `intersect` requires that all relations have the same number of columns. For example, postgres would fail with `ERROR:  each INTERSECT query must have the same number of columns`. DataFusion now has a check for this.